Sources/Experiments/ExPlat.swift

```diff
@@ -100,6 +111,18 @@ import Cocoa
             return .treatment(variation)
         }
     }
+    
+    /// Checks if the experiment name is contained in the enrolled experiments array
+    /// returns false if there is no dictionary or the experiment is not registered
+    ///
+    public func isEnrolled(_ name: String) -> Bool {
```

Comment:
Could we call this `isEnrolled(in name: String)`?

Reply:
totally